sqlmesh/core/context.py

```diff
@@ -743,7 +743,7 @@ def config_for_path(self, path: Path) -> Config:
                 pass
         return self.config
 
-    def config_for_node(self, node: str | Model | StandaloneAudit) -> Config:
+    def config_for_node(self, node: str | Model | Audit) -> Config:
```

Comment:
why was this one changed? i don't think regular audits should be passed in here right?

Reply:
So I changed this because otherwise I get the following error:
```
sqlmesh/core/context.py:895: error: Argument 1 to "config_for_node" of "GenericContext" has incompatible type "Union[SqlModel, SeedModel, PythonModel, ExternalModel, ModelAudit, StandaloneAudit]"; expected "Union[str, SqlModel, SeedModel, PythonModel, ExternalModel, StandaloneAudit]"  [arg-type]
sqlmesh/core/context.py:905: error: Argument 1 to "config_for_node" of "GenericContext" has incompatible type "Union[SqlModel, SeedModel, PythonModel, ExternalModel, ModelAudit, StandaloneAudit]"; expected "Union[str, SqlModel, SeedModel, PythonModel, ExternalModel, StandaloneAudit]"  [arg-type]
tests/core/test_snapshot_evaluator.py:2676: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
Found 2 errors in 1 file (checked 414 source files)
```
But maybe the correct way to address this would be changing [this line](https://github.com/TobikoData/sqlmesh/pull/3131/files#diff-02c2689e9e8516b4f905e12fa4dac9b0fff86efae7964c29fb9337055f6f3d94R889) to
```python
        format_targets = {**self._models, **self._standalone_audits}
```
WDYT?

Reply:
how are you getting this error? are you using mypy?

Reply:
Yes, `mypy`. Are you not able to recreate on your end?

Reply:
To be clear, I'm just running `make style` locally.

Reply:
Let me push with the change reverted and see if it shows up in your CI checks.

Reply:
Yep. Same error in your CI now:
<img width="1352" alt="Screenshot 2024-09-16 at 10 13 06" src="https://github.com/user-attachments/assets/0bc94768-e05c-46af-a0e1-6443c6b74015">

Reply:
ok let's add it back

Reply:
restored